docs/releases/handling-releases.md

```diff
@@ -1,219 +1,33 @@
 # Handling Releases
 
-This document (and the [related PR template](../../.github/release_pull_request_template.md)) outlines the process of releasing new versions of the blocks plugin.
+The WooCommerce Blocks project has a number of automations that aid with making releases. The following are the steps you should take for making a release:
 
-## Prerequisites - what you need to release WooCommerce Blocks
+* Create a GitHub milestone for the next release.
+* Double-check any merged pulls since the last release that should be included in the release notes are assigned to the milestone for this release. **Note:** You don't need to include renovate pull requests in the milestone.
+* If there are any remaining open issues in the milestone for the release, do a final check with the team to verify none of those *need* to go into the release. After verifying, go through and move any open issues/pulls into the next milestone (except for any that have the `type: blocker` label). After this any `type: blocker` labelled issues must be complete and merged into the release branch (cherry-pick if necessary) before continuing.
+* Use the [GitHub UI](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch) to create a release branch for the version being released. The format for the branch name should be `release/x.x` where `x.x` is the version being released.
```

Comment:
> * Use the [GitHub UI]...

Just curious, is there any benefit on using the GitHub UI over using the command line to create the release branch?

Reply:
The automation fires on the branch/tag `create` event. So far the only way I've seen that triggered is creating the branch via GitHub endpoints (not via git publishing a new branch via command line).

With that said, if you have an alternative method for triggering the `create` event on a new branch, you can definitely use it. For the purpose of the docs I thought I'd include the most reliable mechanism for doing so.